libs/wingsdk/src/target-sim/queue.inflight.ts

```diff
@@ -102,12 +104,27 @@ export class Queue implements IQueueClient, ISimulatorResourceInstance {
             type: TraceType.RESOURCE,
             timestamp: new Date().toISOString(),
           });
-          this.messages.push(...messages);
+          this.pushMessagesBackToQueue(messages);
         });
         processedMessages = true;
       }
     } while (processedMessages);
   }
+
+  public async pushMessagesBackToQueue(messages: any): Promise<void> {
```

Comment:
Can we add a more specific type for `messages`?

Reply:
Sure ✅